### PR TITLE
[docs] Loader docs

### PR DESCRIPTION
**Pull request by stevhliu**
This guide attempts to centralize all the docs related to loading LoRAs, textual inversion embeddings, and DreamBooth checkpoints. Please feel free to let me know if I'm missing anything! Once we have all the loading methods properly documented here, I'll clean up the rest of the docs where these loading methods are mentioned.

### CODE REVIEW

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+specific language governing permissions and limitations under the License.
+-->
+
+# Load adapters
+
+[[open-in-colab]]
+
+There are several [training](../training/overview) techniques for personalizing diffusion models to generate images of a specific subject or images in certain styles. Each of these training methods produce a different type of adapter. Some of the adapters generate an entirely new model, while other adapters only modify a smaller set of embeddings or weights. This means the loading process for each adapter is also different.
+
+This guide will show you how to load DreamBooth, textual inversion, and LoRA weights.
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/loading_adapters.md:19]: The community often mixes DreamBooth and LoRA checkpoints into one and another. It's important to keep in mind that LoRA is a general topic that can be applied to DreamBooth and text-to-image fine-tuning.

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+You can't unfuse multiple LoRA checkpoints so if you need to reset the model to its original weights, you'll need to reload it.
+
+</Tip>
+
+Now you can generate an image that uses the weights from both LoRAs:
+
+```py
+prompt = "A cute brown bear eating a slice of pizza, stunning color scheme, masterpiece, illustration"
+image = pipeline(prompt).images[0]
+```
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/loading_adapters.md:213]: Should we mention the PEFT integration here to encourage the community to take advantage of its nifty features? I think we should because it's feature complete in terms of using multiple LoRAs.

**Review by sayakpaul — COMMENTED** [review]
Looks quite nice!

File: docs/source/en/using-diffusers/loading_adapters.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/loading_adapters.md:19]: Added a Tip callout to the LoRA section to clarify this point

**Comment by stevhliu** [comment]
Hmm not sure why the CI is failing. I've rebased and run `make fix-copies` and it didn't change anything 🤷‍♂️

File: src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion_img2img.py [hunk 3]
```diff
@@ -134,7 +134,6 @@ class AltDiffusionImg2ImgPipeline(
         feature_extractor ([`~transformers.CLIPImageProcessor`]):
             A `CLIPImageProcessor` to extract features from generated images; used as inputs to the `safety_checker`.
     """
-
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion_img2img.py:137]: why is this changed?

File: src/diffusers/pipelines/versatile_diffusion/modeling_text_unet.py [hunk 4]
```diff
@@ -1264,10 +1264,12 @@ def forward(
             deprecate(
                 "T2I should not use down_block_additional_residuals",
                 "1.3.0",
-                "Passing intrablock residual connections with `down_block_additional_residuals` is deprecated         "
-                "               and will be removed in diffusers 1.3.0.  `down_block_additional_residuals` should only"
-                " be used                        for ControlNet. Please make sure use"
-                " `down_intrablock_additional_residuals` instead. ",
+                (
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/versatile_diffusion/modeling_text_unet.py:1267]: Let's maybe leave as is here no?

**Review by patrickvonplaten — COMMENTED** [review]
Docs changes look great! Let's maybe just try to not also change other code files :-)

**Review by sayakpaul — APPROVED** [review]
Sick!

**Review by patrickvonplaten — APPROVED** [review]
